models/users.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
```

Comment:
Remove all semicolons, add spaces between parens and arguments in function calls.